Test specification/Test cases/Milestone_1/Publisher app/1_General.md

```diff
@@ -0,0 +1,45 @@
+### [General_001]
+---
+*  **Test case name:**  _Mapbox access token._
+*  Description: May embed the Mapbox access token in the app.
+*  **Expected result:** _The map is available and adequately presented to the user._
+
+### [General_002]
+---
+*  **Test case name:**  _Creating Trackable ID._
+*  Description: User can to create trackabel ID.
```

Comment:
Actually what we are doing is creating a trackable (soon to be renamed to order) with a user-entered id, which is labeled as `name` in the UI

Reply:
All clear.

Reply:
Resolving on behalf of @davyskiba